package.json

```diff
@@ -89,6 +89,7 @@
     "@loadable/webpack-plugin": "^5.7.1",
     "compression": "^1.7.4",
     "core-js": "^3.4.1",
+    "cross-env": "^6.0.3",
```

Comment:
Shouldn't it be installed as a dev dependency?

Reply:
➕ above

Reply:
Done!